token.go

```diff
@@ -7,10 +7,12 @@ type TokenType string
 // TokenParams is the set of parameters that can be used when creating a token.
 // For more details see https://stripe.com/docs/api#create_card_token and https://stripe.com/docs/api#create_bank_account_token.
 type TokenParams struct {
-	Params   `form:"*"`
-	Bank     *BankAccountParams `form:"bank_account"`
-	Card     *CardParams        `form:"card"`
-	Customer string             `form:"customer"`
+	Params        `form:"*"`
+	Bank          *BankAccountParams `form:"bank_account"`
+	BankAccountId string             `form:"bank_account"`
+	Card          *CardParams        `form:"card"`
+	CardId        string             `form:"card"`
```

Comment:
The addition of `BankAccountId` and `CardId` go a little off-road compared to our standard conventions. Generally when referencing an ID, you just do it with the name of the target resource (e.g. passing `Plan` when creating a subscription).

I wonder if we should add an `ID` field into the existing `BankAccountParams` and `CardParams` instead? Is the reason you didn't try that because the encoding implementation on these two types of params so messed up already?